### ISSUE TITLE
RegionDisplay without table

### ISSUE THREAD

**Issue by tischi** [mobie/mobie-viewer-fiji#970]
@constantinpape @martinschorb 

I found that the MoBIE Java code can handle a `RegionDisplay` that does not have a table.

It can simply create a table on the fly, using the `sources` field in `RegionDisplay`:

```
final StringColumn regionIDs = StringColumn.create( ColumnNames.REGION_ID, regionDisplay.sources.keySet() );
table = Table.create( regionDisplay.getName() );
table.addColumns( regionIDs );
```

This table will then, of course, not contain any interesting data, but it will serve to navigate to the images by clicking on the table rows, which is very useful by itself, e.g., for a HCS plate data. In addition, one could use the table to annotate images.

TL;DR
I don't know the status of the spec, but I think we could say that `regionDisplay.tableSource` is _optional_.

**Comment by constantinpape** [comment]
> TL;DR I don't know the status of the spec, but I think we could say that `regionDisplay.tableSource` is _optional_.

It's currently mandatory in the spec, but we can change it to optional.

**Comment by tischi** [comment]
Ok, let's make it optional then.

**Comment by constantinpape** [comment]
Done.
